### ISSUE TITLE
DOM::Tiny to-json clashes with JSON::Fast to-json

### ISSUE THREAD

**Issue by lukasvalle** [zostay/p6-DOM-Tiny#12]
Loading Dom::Tiny and JSON::Fast does lead to an error:

`perl6 -e 'use DOM::Tiny; use JSON::Fast;'
===SORRY!=== Error while compiling -e
Cannot import symbol &to-json from JSON::Fast, because it already exists in this lexical scope
at -e:1`

Maybe using labeled export for to-json could help, but would probably breaking module for some of the current users. Maybe there are other solutions to work around name clashes.

**Comment by zoffixznet** [comment]
(just a note to anyone else encountering this bug: just use `need` instead of `use`: 

```bash
$ perl6 -e 'need DOM::Tiny; use JSON::Fast; say DOM::Tiny.parse("<b>Hi</b>").all-text; say to-json [<a b c>]'
Hi
[
  "a",
  "b",
  "c"
]
```

**Comment by zostay** [comment]
Should be fixed in 19612937d660550b1555fecc9ffbc8b1fd37f330

I will release this shortly, assuming CI passes it on.
